### PR TITLE
Todo-List-Test-Part-2

### PR DESCRIPTION
**Pull request by gergitokusse**
### 👇 The following feature was added to the branch. 👇 

- ➡️ New test feature file Interaction.js function.
- ➡️ Test for updated completed task status.
- ➡️ Test for Clear all completed tasks.
- ➡️ Test for editing task description.

### CODE REVIEW

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @gergitokusse, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has commits from only one collaborator. Since this is a pair programming activity, kindly ensure your coding partner makes some meaningful commits towards the project

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/Interaction.test.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+describe('Test for for updating an items completed', () => {
+  test('Testing task status', () => {
+    document.body.innerHTML = '<div class = "todo-container" >'
+    + '<div>'
+      + '<div class = "todo-list">'
+        + '<div id = 1 class = "mydiv">'
+          + '<span class = "desc1">'
+            + '<input type = "checkbox" id = "check1" name = "checking" />'
+            + '<span id = "span21" class = "show" > Task-1</span>'
```
Comment (omar-muhamad) [inline src/Interaction.test.js:14]: - [ ] [Optional][Highly recommended] Its consedered a good practice and good work convention if you include all test files `[...].test.js` and `__mock__` directory inside one folder and you can call it `tests`. so the file structure will look like that:
```
/src
    - /tests
               - interaaction.test.js
               - __mock__
               - storage.test.js
```

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi Team,
#### Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

### Congratulations! 🎉 
### ⚠️Please consider the Optional suggestions below⚠️

robert-redford-nod

## (Highlights) Good Points: :+1:
- Your work is well documented. ✔
- The correct workflow is used.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- Please check the inline comments below ⬇.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/Interaction.test.js (hunk 1 above)
Comment (gergitokusse) [reply src/Interaction.test.js:14]: ### Okay, 🙏🙏🙏 Thank you so much we will do it and it is also better for file management. thank you again for your good comments🙏🙏.

**Pull request merged**